src/common.cpp

```diff
@@ -30,12 +30,14 @@
 inline namespace CUPLA_ACCELERATOR_NAMESPACE
 {
 
+CUPLA_HEADER_ONLY_FUNC_SPEC
```

Comment:
after pulled from your branch on dev with that subsequent merge, this flag was removed leading to multiple definition error, so we have to take care for the merge part.

Reply:
I will rebase this code to be sure my changes are in.

Reply:
maybe it was for the function `cuplaGetErrorString` which was introduced in a separate PR.